common/src/stack/pylib/stack/wizard.py

```diff
@@ -247,6 +247,7 @@ def validateNetwork(self, tup, config_net=False):
 					self.data.Kickstart_PrivateAddress + '/' +
 					self.data.Kickstart_PrivateNetmask,
 					strict=False)
+				ipaddress.ip_address(str(self.data.Kickstart_PrivateGateway))
```

Comment:
The underlying fix is absolutely correct.  The following is just about style:

So it seems like all of the below `self.data.* = str(some_ip_addr_call())` are sort of redundant with the ones just before the `if` statement?  I'd suggest either following the redundant code pattern here and re-assign (matching existing style), or changing the other lines as well (since the assignment appears to be irrelevant - we just want to catch any errors) to follow your pattern.  I guess that would depend - is there a mutation of the data that could happen?  For example, is `1/255.0.0.0` a valid network in the `ipaddress` module?

That's just a quick glance at the surrounding code, however, so I may be mistaken.